docs/ui/foundations/icons.tsx

```diff
@@ -0,0 +1,198 @@
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M19.9274 5.44938C19.9769 5.47373 20.0143 5.51712 20.0312 5.56962L24.3241 18.9041C24.3584 19.0107 24.3001 19.125 24.1937 19.1599L21.8383 19.9305C21.7865 19.9475 21.73 19.9429 21.6816 19.9179C21.6331 19.8928 21.5968 19.8494 21.5806 19.7973L17.4519 6.47918C17.4197 6.37516 17.475 6.26419 17.5775 6.22738L19.7687 5.44044C19.8206 5.4218 19.8779 5.42503 19.9274 5.44938ZM17.8984 6.54464L21.9082 19.4794L23.8746 18.836L19.7083 5.89464L17.8984 6.54464Z"
+        fill={stroke}
+      />
+      <path
+        d="M8.49884 11.6915L19.8375 5.63208L24.1303 18.9665L12.7917 25.0259L8.49884 11.6915Z"
+        fill={fill}
+      />
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M19.9137 5.44331C19.9697 5.46593 20.0127 5.51219 20.0312 5.56962L24.324 18.9041C24.3542 18.9977 24.313 19.0995 24.2262 19.1459L12.8876 25.2053C12.8344 25.2337 12.7713 25.2371 12.7154 25.2145C12.6595 25.1919 12.6164 25.1456 12.598 25.0882L8.30511 11.7537C8.27495 11.6601 8.31613 11.5583 8.40291 11.5119L19.7415 5.45249C19.7947 5.42406 19.8578 5.42069 19.9137 5.44331ZM8.74464 11.7908L12.9107 24.7314L23.8845 18.867L19.7184 5.92635L8.74464 11.7908Z"
+        fill={stroke}
+      />
+      <path
+        d="M8.49888 11.6915L6.30768 12.4784L17.6463 6.41901L19.8375 5.63208L8.49888 11.6915Z"
+        fill={fill}
+      />
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M20.0235 5.54945C20.0668 5.64692 20.0275 5.76126 19.9334 5.81152L8.5948 11.8709C8.58606 11.8756 8.57699 11.8796 8.56767 11.8829L6.37647 12.6699C6.2761 12.7059 6.16494 12.6584 6.12167 12.5609C6.0784 12.4635 6.1177 12.3491 6.21176 12.2988L17.5504 6.23946C17.5591 6.23479 17.5682 6.23077 17.5775 6.22742L19.7687 5.44049C19.8691 5.40444 19.9802 5.45198 20.0235 5.54945Z"
+        fill={stroke}
+      />
+      <path
+        d="M14.9463 10.606L14.1811 10.9967L13.4172 11.3875L12.7985 20.6434L14.3018 21.315L15.447 13.9097L21.4399 17.6666L21.9202 15.9815L14.9463 10.606Z"
+        fill={stroke}
+      />
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M14.8537 10.4247C14.9238 10.389 15.0082 10.3968 15.0705 10.4448L22.0444 15.8204C22.1104 15.8712 22.1387 15.9572 22.1159 16.0373L21.6356 17.7225C21.618 17.784 21.5725 17.8337 21.5128 17.8567C21.453 17.8796 21.386 17.8731 21.3317 17.8391L15.6008 14.2464L14.5029 21.3462C14.4932 21.4089 14.4549 21.4635 14.3991 21.4938C14.3434 21.5241 14.2767 21.5268 14.2188 21.5009L12.7155 20.8293C12.6377 20.7945 12.5897 20.7149 12.5954 20.6299L13.2141 11.3739C13.2189 11.3025 13.2608 11.2389 13.3245 11.2063L14.8537 10.4247ZM13.6126 11.5161L13.011 20.5155L14.1414 21.0205L15.2458 13.8786C15.2563 13.811 15.3001 13.7531 15.3623 13.7246C15.4246 13.6962 15.497 13.7009 15.555 13.7373L21.3184 17.3503L21.6866 16.0585L14.9241 10.8458L13.6126 11.5161Z"
+        fill={stroke}
+      />
+    </svg>
+  );
+};
+
+export const FeatureIcon = (props: IconProps) => {
```

Comment:
This is exported from the styleguide as `<SparklesIcon />`, so we can remove this